tests/test-customize-changesets-controller.php

```diff
@@ -59,7 +114,21 @@ public function test_register_routes() {
 	 * @covers WP_REST_Customize_Changesets_Controller::get_item_schema()
 	 */
 	public function test_get_item_schema() {
-		$this->markTestIncomplete();
+
+		// @todo Add all properties.
+		$request = new WP_REST_Request( 'OPTIONS', '/customize/v1/changesets' );
+		$response = $this->server->dispatch( $request );
+		$data = $response->get_data();
+		$properties = $data['schema']['properties'];
+
+		$this->assertEquals( 7, count( $properties ) );
+		$this->assertArrayHasKey( 'author', $properties );
+		$this->assertArrayHasKey( 'date', $properties );
+		$this->assertArrayHasKey( 'date_gmt', $properties );
+		$this->assertArrayHasKey( 'settings', $properties ); // Instead of content.
+		$this->assertArrayHasKey( 'slug', $properties );
+		$this->assertArrayHasKey( 'status', $properties );
+		$this->assertArrayHasKey( 'title', $properties );
```

Comment:
It seems that this response is purposefully not returning the sanitize callbacks when using the OPTIONS request since these are not meant for public. Changed that to directly using `get_item_schema` for checking the callbacks.